src/main/groovy/org/pih/warehouse/api/StockMovement.groovy

```diff
@@ -361,6 +376,7 @@ class StockMovement implements Validateable{
                 driverName: shipment.driverName,
                 trackingNumber: trackingNumber?.identifier,
                 comments: shipment.additionalInformation,
+                shipmentComments: shipment.comments,
```

Comment:
If possible, I would like this to replace the "comments" property at some point. We can move "comments" to "additionalInformation" or try to figure out how we might allow the additionalInformation to be come an instance within the "comments" list. This is where we'd need to have a way to classify comments as we'd only ever want one of these "additionalInformation" comments to be added to stockMovement.comments.